src/Parser/HolyPriest/Modules/PriestCore/MasteryBreakdown.js

```diff
@@ -136,16 +142,31 @@ class MasteryBreakdown extends Module {
         this._maxHealVal[tId] = {};
       }
 
+      let eh = event.amount + (event.absorbed || 0) + (event.overheal || 0);
+      if (spellId === SPELLS.RENEW.id) {
+        // due to renew only applying off of initial tick and not from periodic ticks
+        // and both of them having identical spell IDs, I chose to negate renew's mastery
+        // value since the current meta leads us towards never hard-casting renew anyway
+        // which means the mastery value for renew would be miniscule anyway. We uninclude
+        // the value in output for this reason as well.
+
+        // (but I want to include it anyway, that way I can get the "unknown spell" warning to
+        //  work optimally and let users report unknown spells)
+
+        eh = 0;
+      }
+
       if (!(spellId in this._healValByTargetId[tId])) {
-        this._healValByTargetId[tId][spellId] = event.amount + (event.absorbed || 0) + (event.overheal || 0);
+        this._healValByTargetId[tId][spellId] = eh;
       } else {
-        this._healValByTargetId[tId][spellId] += event.amount + (event.absorbed || 0) + (event.overheal || 0);
+        this._healValByTargetId[tId][spellId] += eh;
       }
       this._maxHealVal[tId][spellId] = this._healValByTargetId[tId][spellId];
     }
   }
 
   statistic() {
+    const percOTHD = this.owner.getPercentageOfTotalHealingDone(this.healing);
```

Comment:
Please use descriptive var naming.

Reply:
I felt it was fairly descriptive considering it's an acronym for the method call right after, but I'll tweak it.